### ISSUE TITLE
Language detection not working for some languages

### ISSUE THREAD

**Issue by blakek** [axa-group/nlp.js#99]
**Describe the bug**

Some languages are not working with language detection that should.  The problem seems to be [`franc`](https://github.com/wooorm/franc) is using one set of language codes and this package is using another.

**To Reproduce**

```js
const { Language } = require('node-nlp');

const language = new Language();
console.log(language.guessBest('你叫什么名字？')); // Returns `undefined`
```

**Expected behavior**

 Should show the language is Chinese.

**Additional context**

`zh` or `zho` is language code used in `lib/language/languages.json`, but [`franc`](https://github.com/wooorm/franc) uses `cmn` to represent Mandarin Chinese.  Updating the 3-character code to match that from `franc` seems to work.

**Comment by jesus-seijas-sp** [comment]
Bug confirmed, 
The list of franc codes: https://github.com/wooorm/franc/tree/71b741d00445cbf232fc59c789cb289cfd0d2449/packages/franc-all
Our list of codes:
https://github.com/axa-group/nlp.js/blob/master/lib/language/languages.json

Two possible aproachs:
1. Replace all alpha3 with the franc code
2. Add a field franc in our list containing the franc code and fix this line: https://github.com/axa-group/nlp.js/blob/master/lib/language/language.js#L93

**Comment by blakek** [comment]
Is there a preference for a certain approach?  I'd like to implement this (unless someone else is going already).  I would expect to use `zh` instead of `cmn`.  However, I can see it being aggravating maintaining a lookup to keep up with an external library.

**Comment by jesus-seijas-sp** [comment]
NLP.js internally uses the iso2, not the iso3, so replacing the iso3 with the franc code has no impact. The only impact is that if you only want to use the Language module, the iso3 that will be returned will not be the correct one. 
I don't know if there is people using only this module, instead of franc, to be honest. So for me both aproaches are valid.

I saw the PR, unfortunatelly there seems to be a breaking change from microsoft, and 5 tests crashed. I can accept the PR and fix the version to 1.1.3.
